services/src/test/scala/cromwell/services/cost/GcpCostCatalogServiceSpec.scala

```diff
@@ -80,9 +80,9 @@ class GcpCostCatalogServiceSpec
 
```

Comment:
We need to add unit tests to this file that actually use GPUs, not just make the compiler happy by updating to the new types that support GPUs (as I did).